client/filter/tvs.density.js

```diff
@@ -132,8 +144,19 @@ export function updateTempRanges(xscale, s, range, inputRange, minvalue, maxvalu
 	}
 }
 
+/** Set the start and stop displayed to the user
+ * Encapsulated and exported to verify usage via testing and CI.
+ */
+
+export function setStartStopDisplays(range, inputRange) {
```

Comment:
explain these two params

Reply:
I added comments in [6c25f4d](https://github.com/stjude/proteinpaint/pull/2439/commits/6c25f4d3a9ca5b584010b5d3a3aad0d0d2fe761b)